container-scan/action.yaml

```diff
@@ -1,152 +1,78 @@
-name: "Security Scan Composite Action"
-description: "GitHub Action for scanning container image for vulnerabilities using Lacework"
+name: "Container Security Scan"
+description: "Scans container images for vulnerabilities using Lacework"
 
 inputs:
-  dockerhub-user:
+  image-name:
     required: false
-    default: ""
-    description: username for dockerhub
-  dockerhub-password:
+    description: "Docker image name"
+  image-tag:
+    required: true
+    description: "Docker image tag"
+  enable-docker-build:
     required: false
-    default: ""
-    description: password for dockerhub
-  docker-config-file:
```

Comment:
You are deleting this input, which means this should be a breaking change.

Reply:
Although I think you should just keep this input and pass it through to the docker build action. See https://github.com/open-turo/actions-security/blob/main/docker-build/action.yaml#L5